ansible_collections/arista/avd/roles/eos_validate_state/python_modules/tests/avdtestinterfaces.py

```diff
@@ -60,10 +60,9 @@ def generate_test_details(interface: dict, description_template: str) -> tuple(s
             """
             if "Loopback" in description_template:
                 return "up", "up", description_template
-            elif interface["shutdown"]:
+            if interface["shutdown"]:
```

Comment:
a loopback could technically be shutdown:

```
host1(config-if-Et1)#show int  desc  | json
{
    "interfaceDescriptions": {
        "Ethernet1": {
            "lineProtocolStatus": "down",
            "description": "edge1-Ethernet4",
            "interfaceStatus": "adminDown"
        },
        "Loopback666": {
            "lineProtocolStatus": "down",
            "description": "",
            "interfaceStatus": "adminDown"
        },
```

and we do support it in structured_config: https://avd.sh/en/stable/roles/eos_cli_config_gen/docs/input-variables.html#loopback-interfaces

Reply:
(while we are fixing shutdown issues..)